### PR TITLE
[BUGFIX release] ease core-object upgrade for addons which lack .project

### PR DESCRIPTION
**Pull request by stefanpenner**

### CODE REVIEW

**Comment by Turbo87** [comment]
👍

File: tests/unit/models/addon-test.js [hunk 1]
```diff
@@ -31,6 +31,17 @@ describe('models/addon.js', function() {
     });
   });
 
+  describe('old core object compat', function() {
+    it.only('treeGenerator works without .project', function() {
```
Comment (stefanpenner) [inline tests/unit/models/addon-test.js]: oops

**Comment by stefanpenner** [comment]
this will need to be (once green) cherry-picked to all the branches

- [ ] beta
- [ ] release

**Comment by stefanpenner** [comment]
Confirmed this fixes the issue in question. I'll be able to cut a new release around noon PST

**Comment by nathanhammond** [comment]
@homu r-

**Comment by nathanhammond** [comment]
Blocking for a second: we need to provide an error message or we'll never migrate the community off of broken addons.

**Comment by stefanpenner** [comment]
> Blocking for a second: we need to provide an error message or we'll never migrate the community off of broken addons.

I can add a warning

**Comment by stefanpenner** [comment]
screen shot 2016-06-10 at 11 42 25 am
color is wrong but if

**Comment by stefanpenner** [comment]
Ah my OCD wont allow that, fixed:

screen shot 2016-06-10 at 11 47 49 am

**Comment by nathanhammond** [comment]
@homu r+

**Pull request merged**